modules/nf-commons/build.gradle

```diff
@@ -36,5 +36,7 @@ dependencies {
 
     /* testImplementation inherited from top gradle build file */
     testImplementation(testFixtures(project(":nextflow")))
+    testImplementation("org.codehaus.groovy:groovy-sql:3.0.10")
+    testImplementation('com.h2database:h2:1.4.200')
```

Comment:
Testing a the plugin loading is a good idea, however, I would like to avoid depending the SQL stuff, both in the terms on deps and also because the test logic is quite complicated because requires to run some SQL query. 

Think it would be easier to use the 'nf-hello' plugin https://github.com/nextflow-io/nf-hello

Reply:
The main "problem" is if I'm not wrong, `nf-hello' plugin doesn't have any function to import and test

Reply:
Let's add it, just commented